bodhi/server/models.py

```diff
@@ -1149,6 +1153,10 @@ class Update(Base):
 
     user_id = Column(Integer, ForeignKey('users.id'))
 
+    # Greenwave
+    greenwave_policies_satisfied = Column(Boolean, default=True, nullable=False)
```

Comment:
@bowlofeggs , to be clear, if Greenwave says "yes", should I mark the ci_status field "passed" for all the builds of the update? As you have mentioned before, you want to move the ci_status field from the Build model to the Update model, have you started yet? If not. I can add this as a part of this PR.